pkg/monitor/cluster/version.go

```diff
@@ -0,0 +1,45 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	configclient "github.com/openshift/client-go/config/clientset/versioned"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/Azure/ARO-RP/pkg/util/restconfig"
+)
+
+func (mon *Monitor) emitClusterVersion() error {
+	restConfig, err := restconfig.RestConfig(mon.env, mon.oc)
+	if err != nil {
+		return err
+	}
+
+	configCli, err := configclient.NewForConfig(restConfig)
+	if err != nil {
+		return err
+	}
```

Comment:
move to NewMonitor.  Rationale: will simplify unit test.

Reply:
I've done this, not sure if it is exactly what you meant however.